testplan/testing/multitest/base.py

```diff
@@ -1222,6 +1230,9 @@ def _run_testcase(
                     )
 
                 time_restriction = getattr(testcase, "timeout", None)
+                # Use default testcase timeout from config if no explicit timeout is set
+                if time_restriction is None and self.cfg.testcase_timeout:
```

Comment:
The problem here is in our tests, multitest doesn't always have a parent, thus when timeout is not provided at multitest level, self.cfg.testcase_timeout could raise no such attr. We can use hasattr/getattr here.